src/mon/Paxos.cc

```diff
@@ -358,6 +358,7 @@ bool Paxos::store_state(MMonPaxos *m)
     dout(10) << "store_state ignoring all values, they start at " << start->first
 	     << " > last_committed+1" << dendl;
     start = m->values.end();
+    return changed;
```

Comment:
if we want to early return here, maybe we can remove the line of `start = m->values.end();` also.